virtualizarr/readers/hdf/hdf.py

```diff
@@ -371,6 +371,10 @@ def _virtual_vars_from_hdf(
             group_name = "/"
 
         variables = {}
+        non_coordinate_dimesion_vars = HDFVirtualBackend._find_non_coord_dimension_vars(
+            group=g
+        )
```

Comment:
nitpick: typo (`dimesion` -> `dimension`)
```suggestion
        non_coordinate_dimension_vars = (
            HDFVirtualBackend._find_non_coord_dimension_vars(group=g)
        )
```